WebJobs.proj

```diff
@@ -124,8 +124,36 @@
       </Binplace>
     </ItemGroup>
   </Target>
+    
+  <Target Name="UpdateAssemblyInfo">   
```

Comment:
I assume you're going to apply the same changes to the other 2 repos so we're consistent (extensions, script)?

Reply:
Yep. I got most of this from Script -- except it doesn't do the commit stamping. I'll move that over. And yeah, I'll do this to extensions as well.